src/receivers/HTTPReceiver.ts

```diff
@@ -4,6 +4,7 @@ import { ListenOptions } from 'net';
 import { parse as qsParse } from 'querystring';
 import { Logger, ConsoleLogger, LogLevel } from '@slack/logger';
 import { InstallProvider, CallbackOptions, InstallProviderOptions, InstallURLOptions } from '@slack/oauth';
+import { URL } from 'url';
```

Comment:
This is not related to this issue but I found that tsd with the latest type definition fails with the following error:

```
> @slack/bolt@3.3.0 test:types /Users/ksera/github/bolt-js
> tsd


  src/receivers/HTTPReceiver.ts:237:35
  ✖  237:35  Cannot find name URL.

  1 error

npm ERR! code ELIFECYCLE
```